### DISCUSSION TITLE
go Directly to books/new (tab child)

### DISCUSSION THREAD

**Discussion by fullflash** [csells/go_router#378] [Q&A]
Hi,
in your example books app how can we acchive to directly open sub route child tab ?

it would be nice if we can directly open nested route page there are a gg_route package with example to achive this.

i have tried adding extra textbutton in your settings page 
but it just opens books main page first tab now the last one as it should be for path "/books/all
```
 TextButton(
              onPressed: () {
                context.go('/books/all');
              },
              child: const Text('Go directly to /books/all (GoRouter)'),
            ),
```

**Comment by csells** [comment]
I'm afraid I'm not understanding you question. What are you trying to do? What code are you using? What is the expected and actual behavior?
